CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Piral Changelog
 
-## 0.9.3 (tbd)
+## 0.9.3 (December 17, 2019)
```

Comment:
Please don't change this. The date is auto generated on release.

Reply:
ok - thanks for highlighting